From d93a71665390f5d1956fcf445457ba816f1a2e6d Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Sat, 19 Jul 2014 04:22:57 -0400
Subject: [pci] Use dev_is_pci() to identify PCI devices

Message-id: <20140719042256.19438.28414.stgit@gir.stowe>
Patchwork-id: 85964
O-Subject: [RHEL7.1 PATCH 02/81] PCI: Use dev_is_pci() to identify PCI devices
Bugzilla: 1110895
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110895
Upstream Status: 40c368c1ef6379db08d2ebd36f5a45efb93cff73

commit 40c368c1ef6379db08d2ebd36f5a45efb93cff73
Author: Yijing Wang <wangyijing@huawei.com>
Date:   Thu Dec 5 19:52:53 2013 +0800

    PCI: Use dev_is_pci() to identify PCI devices

    Use dev_is_pci() instead of checking bus type directly.

    Signed-off-by: Yijing Wang <wangyijing@huawei.com>
    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/pci/pci-acpi.c b/drivers/pci/pci-acpi.c
index b2d8cba..988e75f 100644
--- a/drivers/pci/pci-acpi.c
+++ b/drivers/pci/pci-acpi.c
@@ -359,7 +359,7 @@ static void pci_acpi_cleanup(struct device *dev)
 
 static bool pci_acpi_bus_match(struct device *dev)
 {
- return dev->bus == &pci_bus_type;
+ return dev_is_pci(dev);
 }
 
 static struct acpi_bus_type acpi_pci_bus = {
-- 
1.7.1